services/utils.js

```diff
@@ -0,0 +1,39 @@
+const NEXT_PUBLIC_API_URL = process.env.NEXT_PUBLIC_API_URL;
+const NEXT_PUBLIC_GET = process.env.NEXT_PUBLIC_GET;
```

Comment:
Get and post dont need to be environment variables. We can use class constants here